### PR TITLE
Remove defaults and use runtime-tools/generate for spec 

### PR DESCRIPTION
**Pull request by rhatdan**
After Capabilities get merged, we need to switch to using opencontainers/runtime-tools/generate

### CODE REVIEW

**Comment by mheon** [comment]
No objection to moving to a library to do spec generation, though I think @baude should definitely look this over

File: cmd/kpod/spec.go [hunk 1]
```diff
@@ -110,8 +126,20 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 		configSpec.Linux.Seccomp = &seccompConfig
 	}
 
+	configSpec.Process.Env = config.env
```
Comment (mheon) [inline cmd/kpod/spec.go]: Does the spec generation have a way to add environment and bind/tmpfs mounts? If so maybe we should add a TODO to convert to using that at some point. I don't think we need to migrate to it right away.

Comment (rhatdan) [reply cmd/kpod/spec.go]: Yes, I will convert those over as well.

**Comment by baude** [comment]
I looked it over this weekend, LGTM.

**Comment by baude** [comment]
@rh-atomic-bot r+ 32ec389

**Pull request closed without merging**